test-suite/shared/src/test/scala/org/scalajs/testsuite/niocharset/CharsetTest.scala

```diff
@@ -67,4 +70,24 @@ class CharsetTest {
 
     assertFalse(Charset.isSupported("this-charset-does-not-exist"))
   }
+
+  @Test def aliases(): Unit = {
+    assertEquals(Charset.forName("UTF-8").aliases(),
+      javaSet("UTF8", "unicode-1-1-utf-8"))
+    assertEquals(Charset.forName("UTF-16").aliases(),
+      javaSet("UTF_16", "unicode", "utf16", "UnicodeBig"))
+    assertEquals(Charset.forName("UTF-16BE").aliases(),
+      javaSet("X-UTF-16BE", "UTF_16BE", "ISO-10646-UCS-2",
+          "UnicodeBigUnmarked"))
```

Comment:
Here, you'll have 4 spaces of indentation on top of 4 spaces of indentation, so at the end of the day those lines should look like:
```scala
    assertEquals(Charset.forName("UTF-16BE").aliases(),
        javaSet("X-UTF-16BE", "UTF_16BE", "ISO-10646-UCS-2",
            "UnicodeBigUnmarked"))
```